app/src/main/java/org/simple/clinic/removeoverdueappointment/RemoveOverdueUiRenderer.kt

```diff
@@ -0,0 +1,20 @@
+package org.simple.clinic.removeoverdueappointment
+
+import org.simple.clinic.contactpatient.RemoveAppointmentReason
+import org.simple.clinic.mobius.ViewRenderer
+
+class RemoveOverdueUiRenderer(private val ui: RemoveOverdueUi) : ViewRenderer<RemoveOverdueModel> {
+
+  override fun render(model: RemoveOverdueModel) {
+    val allRemoveAppointmentReasons = RemoveAppointmentReason.values().toList()
```

Comment:
Since this list will never change, should we make it a class property instead of a local variable?

Reply:
Yeah, good point. Will make the change.